integration/test/appcatalog/appcatalogentry/app_catalog_entry_test.go

```diff
@@ -102,19 +102,19 @@ func TestAppCatalogEntry(t *testing.T) {
 
 		expectedEntrySpec := v1alpha1.AppCatalogEntrySpec{
 			AppName:    "prometheus-operator-app",
-			AppVersion: "0.38.1",
+			AppVersion: "0.43.2",
```

Comment:
Won't this break again when we create more releases? What about using the appcatalog library to get the latest release from the catalog?

Reply:
Good catch, sure, it would be better to pull the latest chart from the appcatalog each time.